server/src/com/thoughtworks/go/server/service/SecurityAuthConfigService.java

```diff
@@ -57,4 +62,34 @@ public void create(Username currentUser, SecurityAuthConfig securityAuthConfig,
         update(currentUser, securityAuthConfig, result, new SecurityAuthConfigCreateCommand(goConfigService, securityAuthConfig, authorizationExtension, currentUser, result));
     }
 
+
+    public void verifyConnection(SecurityAuthConfig securityAuthConfig, LocalizedOperationResult result) {
```

Comment:
Check if the plugin has verification capability, and if not throw a 400 status code?